atacworks/dl4atac/utils.py

```diff
@@ -399,3 +416,137 @@ def log(self, names, normalizer=1.0, reset=True):
                 reset=reset) * 1000.0 / normalizer
             string += ' | {}: {:.2f}'.format(name, elapsed_time)
         myprint(string)
+
+
+def _get_tiling_intervals(sizes, intervalsize):
+    """Produce intervals of given chromosomes.
+
+    Tile from start to end of given chromosomes, shifting by given length.
+
+    Args:
+        sizes: Pandas df containing columns 'chrom' and 'length',
+            with name and length of required chromosomes
+        intervalsize: length of intervals
+        shift: distance between starts of successive intervals.
+
+    Returns:
+        Pandas DataFrame containing chrom, start, and end of tiling intervals.
+
+    """
+    # Default: non-overlapping intervals
+    shift = intervalsize
```

Comment:
We can drop this shift parameter since we don't use it.

Reply:
done